src/controllers/openid4vc/issuance-sessions/issuance-sessions.service.ts

```
@@ -0,0 +1,156 @@
+import { OpenId4VcIssuanceSessionRepository } from '@credo-ts/openid4vc/build/openid4vc-issuer/repository'
+
+import { SignerMethod } from '../../../enums/enum'
+import { BadRequestError, NotFoundError } from '../../../errors/errors'
+import { X509_CERTIFICATE_RECORD } from '../../../utils/constant'
+import { Request as Req } from 'express'
+
+class IssuanceSessionsService {
+  public async createCredentialOffer(
+    options: OpenId4VcIssuanceSessionsCreateOffer,
+    agentReq: Req
+  ) {
```

ESLint

[error] Replace `⏎····options:·OpenId4VcIssuanceSessionsCreateOffer,⏎····agentReq:·Req⏎··` with `options:·OpenId4VcIssuanceSessionsCreateOffer,·agentReq:·Req`

(prettier/prettier)